pkg/kubelet/stats/cri_stats_provider_windows.go

```diff
@@ -51,7 +51,9 @@ func (p *criStatsProvider) listContainerNetworkStats() (map[string]*statsapi.Net
 			klog.V(4).Infof("Failed to get statistics for container %q with error '%v', continue to get stats for other containers", c.ID, err)
 			continue
 		}
-
+		if err := container.Close(); err != nil {
```

Comment:
Does this need to be part of the error handling path? E.g. if `container.Statistics()` gets an error, it looks like we'll skip closing the handle. 

Might better to use a defer (but if you do please test it to ensure `container` is tracked properly across loop iterations, I always have trouble remembering whether receivers are evaluated at defer time or return time).

Reply:
Agree. We should probably defer after handling the error of OpenContainer (~ line 48).

Reply:
Ahh, quite true.

Reply:
@mtaufen @yujuhong I moved it to a private method so I could use `defer`, the only trouble is catching the `Close()` error in a defer is now sorta tricky. I used the `go.uber.org/multierr` package since it helps solve this problem nicely, and was already vendored but it's right now a transitive dependency for the project. I don't know if promoting it to a direct dependency is something people worry about. It looks like k8s largely follows the "ignore deferred errors" pattern today 😄.

Reply:
On second thought, and after seeing it triggers a new level of PR review to add that dep, I decided to go with a simpler approach that avoids the new dep :)